tests/test_nn/test_emb.py

```diff
@@ -58,20 +57,15 @@ def _pre_instantiation_hook(self, kwargs: MutableMapping[str, Any]) -> MutableMa
         kwargs = super()._pre_instantiation_hook(kwargs=kwargs)
         # TODO: use triple generation
         # generate random triples
-        mapped_triples = numpy.stack([
-            numpy.random.randint(max_id, size=(self.num_triples,))
+        mapped_triples = torch.stack([
```

Comment:
there's even a function already built to do this triples generation we could use

Reply:
I suppose that is what the TODO comment was asking for :smile: 

```python
# TODO: use triple generation
```

Reply:
yes I think last time I put that there, it was irrelevant for the PR so I didn't want to make it bigger. but this numpy stuff is silly so time to switch it over. when I do the other thing for you I will also take care of this

Reply:
done in 08183ba